sorting/python/quick-sort.py

```diff
@@ -0,0 +1,12 @@
+def quickSort(arr):
```

Comment:
**Just a recommendation :**
 you can put same comments explaining the algorithm along with some basic analysis (time and space complexity) information about quicksort. It will make is easier for the people who are new to this algo understand it a bit better and will also make your program more pleasing.

Reply:
I can add some documentation, just not now. Is it okay if I don that in a day or two? I can also create another PR for the docs after some time if that's okay.